### PR TITLE
a few of my chapters

### PR DESCRIPTION
**Pull request by Ganome**
See commit history for more details

### CODE REVIEW

File: Linux Permissions and Ownership/The sudo Command.md
Comment (jajunk) [inline Linux Permissions and Ownership/The sudo Command.md:1]: Doesn't follow the standard Markdown syntax

Comment (jajunk) [reply Linux Permissions and Ownership/The sudo Command.md:1]: I editied this file to reflect the standard Markdown format. Please review the below link for assistance. 

[MarkDownStyleGuide](https://github.com/jajunk/LightHouseLinuxMasterMDFiles/blob/dev/LightHouseMarkDownStyleGuide.md)

**Review by jajunk — CHANGES_REQUESTED** [review]
@Ganome

I have updated the pull request to target the `dev` branch instead of `main`. Please ensure to direct future pull requests to the `dev` branch as well.

Additionally, I noticed that the markdown guide is not being followed. I have linked the guide in a comment on the first file for your reference.

Could you please confirm if you will be handling both the content generation and the styling, or if we should assign the styling to someone else?

**Comment by Ganome** [comment]
I believe these changes now meet the styling guide.

**Comment by jajunk** [comment]
I will look this over and merge it in the morning. Thank you for making these changes.

**Pull request merged**